src/edu/csus/ecs/pc2/ui/EditProblemPane.java

```diff
@@ -425,19 +425,19 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
      * message "Problem Modified - Save Changes?") and responding "Yes" to the message.
      * 
      */
-    protected void addProblem() {
+    protected boolean addProblem() {
 
         if (problemNameTextField.getText().trim().length() < 1) {
             showMessage("Enter a problem name (\"General\" tab)");
-            return;
+            return false;
         }
 
         if (!validateProblemFields()) {
             // new problem is invalid, just return; message issued by validateProblemFields
```

Comment:
Change comment to indicate `false `is returned, since it no longer just "returns".

Reply:
Agree.